utxonursery.go

```diff
@@ -257,7 +257,16 @@ func (u *utxoNursery) Start() error {
 	// connected block. We register immediately on startup to ensure that
 	// no blocks are missed while we are handling blocks that were missed
 	// during the time the UTXO nursery was unavailable.
-	newBlockChan, err := u.cfg.Notifier.RegisterBlockEpochNtfn()
+	bestHash, bestHeight, err := u.cfg.ChainIO.GetBestBlock()
+	if err != nil {
+		return err
+	}
+	newBlockChan, err := u.cfg.Notifier.RegisterBlockEpochNtfn(
```

Comment:
is this equivalent to just passing in `nil`? down the road we'll want to pull this value from the db and pass it in, though that is better left for a later PR as might require migration